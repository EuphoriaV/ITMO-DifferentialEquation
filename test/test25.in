5.1
((((10.2 + 5.15) - (6.13 * x)) * ((8.53 + (x * 4.85)) * (3.45 * x))) * ((7.31 * ((x * 9.47) * (6.93 * x))) * (((5.62 - 5.7) * (4.15 - 4.41)) + x)))
4.92
4.41