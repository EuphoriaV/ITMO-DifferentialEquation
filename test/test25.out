-0.0015551